crates/forge/tests/cli/script.rs

```diff
@@ -242,10 +242,10 @@ Script ran successfully.
 ==========================
 Simulated On-chain Traces:
 
-  [45299] → new GasWaster@[..]
-    └─ ← [Return] 226 bytes of code
+  [44291] → new GasWaster@0x810090f35DFA6B18b5EB59d298e2A2443a2811E2
```

Comment:
`[..]` is preferred here I think

Reply:
I can revert but this was automatically replaced by running with `SNAPSHOTS=overwrite` and is handy IMO

Reply:
:man_facepalming: yes, indeed, better `[..]`, changed in https://github.com/foundry-rs/foundry/pull/9131/commits/1ecd4d03b792ccae4aa771a8e8c063a5d26cf03f